docs/source/models/cause.rst

```diff
@@ -171,29 +174,321 @@ Common data sources for cause models
+`Non-standard data sources`_ are discussed later on this page.
+
+.. list-table:: Data Definitions
+   :widths: 20 30 30
+   :header-rows: 1
+
+   * - Measure
+     - Definition
+     - Uses
+   * - `Prevalence`_
+     - Proportion of population with a given condition.
+     - Initialize cause model states.
+   * - `Birth Prevalence`_
+     - Proportion of all live births born with a given condition.
+     - Initialize neonatal cause model states.
+   * - `Incidence`_
+     - Number of new cases of a given condition per person-year.
+     - Estimate transition rates.
+   * - `Remission`_
+     - Number of recovered cases from a given condition per person-year.
+     - Estimate transition rates.
+   * - `Disability Weights`_
+     - Proportion of full health not experienced due to disability associated
+       with a given condition.
+     - Measure disability attributed to cause model states.
+   * - `Cause-specific Mortality`_
+     -
+     -
+   * - `Excess Mortality`_
+     -
+     -
 
-Incidence
-+++++++++
+Prevalence
+++++++++++
+
+Prevalence is defined as the **proportion of a given population that possesses
+a given condition or trait** at a given time-point.
 
-Birth prevalence
+  For example, the prevalence of obesity in the United States was
+  approximately 40% in 2016.
+
+When a *time-frame* (such as 2016, i.e. 1/1/16 to 12/31/16) instead of a
+*time-point* (such as 1/1/16) is reported, it is commonly assumed that the
+reported prevalence represents the prevalence of the *midpoint* of
+that time-frame (7/1/16 is the midpoint for the time frame of all of 2016).
+However, this may not always be the case and it should be considered when
+relevant.
+
+Prevalence data can be used to **initialize cause model states** and
+represents the **probability that a simulant will begin the simulation in a
+given state.**
+
+  For example, the probability that a simulant in a model of obesity in the
+  United States beginning in 2016 will begin the simulation as obese is 0.4 or
+  40%.
+
+Birth Prevalence
 ++++++++++++++++
 
-Remission
+Birth prevalence is defined as the **proportion of live births in a given
+population that possess a given condition or trait at birth.**
+
+  For example, the birth prevalence for cleft lip in the United States in 2006
+  was 10.6 per 10,000 live births, or 0.106%.
+
+Birth prevalence data can be used to **initialize neonatal cause model
+states** and represent the **probability that a simulant who is born during
+the simulation will be born into a given neonatal cause model state.**
+
+  For example, the probability that a simulant born during a simulation of
+  cleft lip in the United States in 2006 is 0.00106, or 0.106%.
+
+Incidence
 +++++++++
 
-Prevalence
+Incidence rates are defined as the **number of new cases of a condition that
+occur per person-year of the at-risk population (individuals without
+condition).** Specifically, the at-risk population can be represented as
+`1 - condition prevalence`.
```

Comment:
Doing a little slight of hand with the definition of prevalence here.

Reply:
What exactly is the sleight of hand again?

Reply:
Above we define prevalence as a point estimate and here we are implicitly using it as an annual average.

Reply:
I changed to:  Specifically, the at-risk population at a given time can be 
represented as `1 - condition prevalence at that time`

Reply:
(I changed it locally... remote updates coming soon!)